pyglmnet/utils.py

```diff
@@ -6,6 +6,11 @@
 from copy import copy
 import logging
 
+try:
+    from tqdm import tqdm
+    has_tqdm = True
```

Comment:
@jasmainak this way if tqdm is not found, we can use the default behaviour instead.

Reply:
okay looks reasonable to me @geektoni